EIPS/eip-1363.md

```diff
@@ -184,23 +186,33 @@ interface ERC1363Spender {
 ``` 
 
 ## Rationale
-There are many proposed uses of Ethereum smart contracts that can accept ERC-20 payments (i.e. to create a token payable crowdsale, selling services for tokens, paying invoices, making subscriptions, use them for a specific utility and many other purposes). 
+There are many proposed uses of Ethereum smart contracts that can accept [ERC-20](./eip-20.md) payments. 
+
+Examples could be 
+* to create a token payable crowdsale
+* selling services for tokens 
+* paying invoices
+* making subscriptions
+
+For these reasons it was named as **"Payable Token"**.
+
+Anyway you can use it for specific utilities or for any other purposes who require the execution of a callback after a transfer or approval received.
```

Comment:
This should be moved to Motivation.  Motivation is where you describe why someone might want to implement this specification/standard.  Rationale is where you describe *specific* decisions made in the specification like "why use the constant 5 instead of 7" or "why name the function foo instead of bar".

Note: It is OK to have a mostly empty Rationale section if everything is straightforward and no one expressed any confusion over anything.  I think the comment you have about naming of the functions is enough.  Regarding the name "Payable Token", that isn't actually part of the specification (just the title of the EIP) so it isn't relevant to the rationale section.  :)